doc/user/content/sql/create-source/mysql.md

```diff
@@ -0,0 +1,387 @@
+that overrides `binlog_expire_logs_seconds` and is set to `0` by default.
+
+#### Creating a source
+
+Materialize ingests the raw replication stream data for all (or a specific set
+of) tables in your upstream MySQL database.
+
+```sql
+CREATE SOURCE mz_source
+  FROM MYSQL CONNECTION mysql_connection
+  FOR ALL TABLES;
+```
+
+When you define a source, Materialize will automatically:
+
+1. Create a **subsource** for each original table upstream, and perform an
+   initial, snapshot-based sync of the tables before it starts ingesting change
+   events.
+
+    ```sql
+    SHOW SOURCES;
+    ```
+
+    ```nofmt
+             name         |   type    |  size   |  cluster  |
+    ----------------------+-----------+---------+------------
+     mz_source            | mysql     |         |
+     mz_source_progress   | progress  |         |
+     table_1              | subsource |         |
+     table_2              | subsource |         |
+    ```
+
+1. Incrementally update any materialized or indexed views that depend on the
+   source as change events stream in, as a result of `INSERT`, `UPDATE` and
+   `DELETE` operations in the upstream MySQL database.
+
+It's important to note that the schema metadata is captured when the source is
+initially created, and is validated against the upstream schema upon restart.
+If you create new tables upstream after creating a MySQL source and want to
+replicate them to Materialize, the source must be dropped and recreated.
+
+##### MySQL schemas
+
+`CREATE SOURCE` will attempt to create each upstream table in the same schema as
+the source. This may lead to naming collisions if, for example, you are
+replicating `schema1.table_1` and `schema2.table_1`. Use the `FOR TABLES`
+clause to provide aliases for each upstream table, in such cases, or to specify
+an alternative destination schema in Materialize.
+
+```sql
+CREATE SOURCE mz_source
+  FROM MYSQL CONNECTION mysql_connection
+  FOR TABLES (schema1.table_1 AS s1_table_1, schema2_table_1 AS s2_table_1);
+```
+
+### Monitoring source progress
+
+[//]: # "TODO(morsapaes) Replace this section with guidance using the new
+progress metrics in mz_source_statistics + console monitoring, when available
+(also for PostgreSQL)."
+
+By default, MySQL sources expose progress metadata as a subsource that you
+can use to monitor source **ingestion progress**. The name of the progress
+subsource can be specified when creating a source using the `EXPOSE PROGRESS
+AS` clause; otherwise, it will be named `<src_name>_progress`.
+
+The following metadata is available for each source as a progress subsource:
+
+Field              | Type                                                    | Details
+-------------------|---------------------------------------------------------|--------------
+`source_id_lower`  | [`uuid`](https://materialize.com/docs/sql/types/uuid/)  |
```

Comment:
@rjobanp: I intend to eventually change this section to not use the progress subsource, but instead use `mz_source_statistics` (see TODO). In the meantime, if you have a short way of describing `source_id_lower` and `source_id_upper`, it'd be helpful! I can piece something together from code comments and your blogpost draft later, if nothing comes up.